### PR TITLE
Pypi

### PR DESCRIPTION
**Pull request by somexpert**
This PR is for any changes needed to put the package on pypi and have it be installable view `pip install django-major-event-log`. Once this is closed the package will be uploaded to pypi (currently a version of it is on the test pypi site) and the v0.1.0 milestone will be complete.

### CODE REVIEW

**Comment by somexpert** [comment]
@ldko, @damonkelley, can I get you to give this a one-over?

File: MANIFEST.in [hunk 1]
```diff
@@ -1,3 +1,3 @@
 include LICENSE
 include README.md
-recursive-include major_event_log *.html *.xml *.css *.js *.jpg *.jpeg *.png *.gif
+recursive-include major_event_log/templates *
```
Comment (ldko) [inline MANIFEST.in]: Is it a good idea just to have "*"? should it be *.py?

Comment (somexpert) [inline MANIFEST.in:3]: There shouldn't be any python files in there at all. Those are only going to be .html or .xml files.

Comment (ldko) [inline MANIFEST.in:3]: errr nevermind. yeah, realized that was the templates directory.

**Comment by ldko** [comment]
:+1:

**Comment by damonkelley** [comment]
@somexpert. are we ready for a tag once this is completed?

**Comment by somexpert** [comment]
Yup, we'll complete the milestone and can tag it as version 0.1.0

**Comment by damonkelley** [comment]
Should we add a Python 3 classifier?

**Comment by somexpert** [comment]
python 3, or python 3.4? I haven't tested against any other version

**Comment by damonkelley** [comment]
LTGM :+1:

**Pull request merged**